### DISCUSSION TITLE
Support for Solar Log Inverters

### DISCUSSION THREAD

**Discussion by d28b** [pkuehnel/TeslaSolarCharger#1152] [Show and tell]
Solar Log devices provide the current power via HTTP/REST:
```
POST http://192.168.XX.XX/getjp
```
with data
```
token=;{"780":null,"781":null}
```
The response looks as follows:
```
{"780": 1221, "781": 312}
```
Register 780 reports the generated power in W, while register 781 reports the consumed power in W. The power submitted to the grid (+) or consumed from the grid (-) can be calculated as follows:
```
grid = generated - consumed
```
Since version 2.24.7 of the TeslaSolarCharger supports GET requests only, we use a small bridge which requests the current power values from the Solar Log device as describes above, and provides them via GET requests to TeslaSolarCharger.

Here is the bridge we use:
- [solar-log-bridge.zip](https://github.com/pkuehnel/TeslaSolarCharger/files/14557263/solar-log-bridge.zip)

**Comment by pkuehnel** [comment]
Sorry, missed your post. This will be possible in the next update.
